src/tsickle.ts

```diff
@@ -919,19 +926,26 @@ class Annotator extends ClosureRewriter {
       // inserts an artificial `const prefix = goog.require` call for the module, and then registers
       // all symbols from this import to be prefixed.
       if (!this.options.untyped) {
-        let symbols: ts.Symbol[] = [];
+        let symbols: {name: string, sym: ts.Symbol}[] = [];
         const typeChecker = this.program.getTypeChecker();
         if (importClause.name) {
           // import a from ...;
-          symbols = [typeChecker.getSymbolAtLocation(importClause.name)];
+          symbols = [{
+            name: getIdentifierText(importClause.name),
+            sym: typeChecker.getSymbolAtLocation(importClause.name)
```

Comment:
For renaming imports, I vaguely recall importClause has two members, one which is the name and one which is the rename.  Though maybe the symbol serves for one of those.